### PR TITLE
Skrollr

### PR DESCRIPTION
**Pull request by ethangl**

### CODE REVIEW

File: source/stylesheets/_prose.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  }
+}
+
+#venue {
+  background-color: #d1eacf;
+  background-image: image-url("ritz.png");
+}
+
+#why-austin {
+  background-color: #ffa6c9;
```
Comment (kylefiedler) [inline source/stylesheets/_prose.scss:31]: Should we be putting these in variables?

File: source/stylesheets/_prose.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  }
+
+  h2 {
+    font-size: 5vh;
+    font-weight: 200;
+    line-height: 1.2;
+  }
+
+  p {
+    font-size: 3vh;
```
Comment (kylefiedler) [inline source/stylesheets/_prose.scss:19]: This is hilarious

File: source/stylesheets/_prose.scss (hunk 1 above)
Comment (ethangl) [inline source/stylesheets/_prose.scss:31]: I think they'll probably end up being unique to that section.

**Comment by kylefiedler** [comment]
COOL

File: source/stylesheets/_prose.scss (hunk 2 above)
Comment (ethangl) [inline source/stylesheets/_prose.scss:19]: Hilariously responsive? ✅

**Pull request merged**